how-to/customize-workspace/client/src/framework/workspace/dock.ts

```diff
@@ -182,9 +193,43 @@ export async function minimize() {
 	return Dock.minimize();
 }
 
+export async function getWindowState(): Promise<"normal" | "minimized" | "none"> {
+	if (registrationInfo) {
+		try {
+			const win = fin.Window.wrapSync({ name: "openfin-dock", uuid: "openfin-browser" });
```

Comment:
I don't think we should be doing this. It was done in the past when there was no sdk but we shouldn't be dependant on uuid (it can change) and dock name. This should be a feature request for the api to expose state (e.g. isVisible)

Reply:
Removed